packages/components/pie-webc/src/index.ts

```diff
@@ -1 +1,7 @@
-// Component exports to live in this file
+import { PieButton } from '@justeattakeaway/pie-button';
+import { PieModal } from '@justeattakeaway/pie-modal';
```

Comment:
I'm thinking this means if I call `import '@justeattakeaway/pie-webc;'` then every customElement.define function is called automatically?

I don't necessarily see any issue with this, just curious

Reply:
That's quite likely at the moment. I think this will be investigated fully in the upcoming tree-shaking tickets. It'll be interesting to know if bundlers are clever enough to separate them! If it does manage to split them out then I would imagine that only the imported components are registered.